src/SIL.LCModel.Core/SpellChecking/SpellEngine.cs

```diff
@@ -75,9 +74,10 @@ public bool Check(string word)
 		/// </summary>
 		/// <param name="word"></param>
 		/// <returns></returns>
-		private static byte[] MarshallAsUtf8Bytes(string word)
+		private static string MarshallAsUtf8Bytes(string word)
 		{
-			return Encoding.UTF8.GetBytes(Icu.Normalize(word, Icu.UNormalizationMode.UNORM_NFC) + "\0");
+			byte[] bytes = Encoding.UTF8.GetBytes(Icu.Normalize(word, Icu.UNormalizationMode.UNORM_NFC) + "\0");
```

Comment:
Ok, I now understand, the UTF8 encoding is required, and even though the NHunspell takes a string instead of bytes this is really the only way to get a UTF8 encoded string.